coreos/profiles/ncp.profile

```diff
@@ -0,0 +1,490 @@
+documentation_complete: true
+
+title: 'NIST National Checklist Program Security Guide'
```

Comment:
```suggestion
title: 'NIST National Checklist for Red Hat Enterprise Linux CoreOS'
```

Reply:
Ideally, a version number or range (e.g. 8.x) should be included.

Reply:
Versioning will probably change so much that this would be an exercise in futility.

Reply:
Yeah, you're right, and I see they've been removed for other profiles. Looks good to me!